src/Domain/Entities/Category.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
```

Comment:
Since these files are generated you should add the following to `.gitattributes`:

~~~none
/src/Domain/Entities/* linguist-generated
~~~

Reply:
I heavily edited them after they were generated. I moved the config to individual files, cleaned redundancy, added base classes. 

Honestly I think Ill scrap this PR and just do only generated files and some SQL script

Reply:
Oh, I was not aware that the files were edited.